src/EditorFeatures/Core/InheritanceMargin/AbstractInheritanceMarginService.cs

```diff
@@ -92,8 +92,8 @@ private static bool CanHaveInheritanceTarget(ISymbol symbol)
                 return false;
             }
 
-            if (symbol is INamedTypeSymbol or IEventSymbol or IPropertySymbol ||
-                symbol.IsOrdinaryMethod())
+            if (symbol is INamedTypeSymbol or IEventSymbol or IPropertySymbol
+                or IMethodSymbol { MethodKind: MethodKind.Ordinary or MethodKind.ExplicitInterfaceImplementation })
```

Comment:
Probably will need some tweaks for abstract statics (especially operators)

Reply:
I am not fully tracking the compiler side work. But do you mean this https://github.com/dotnet/csharplang/issues/4436 ? I see it's still at design stage so probably it won't be taken into C# 11.  Is it already implemented?

Reply:
@Cosfine Yes it's in https://github.com/dotnet/roslyn/tree/features/StaticAbstractMembersInInterfaces. Some IDE work (e.g, completion) is being done currently in the feature branch.

Reply:
Then I would prefer first merge main to the feature branch (which I suppose bot would do it regularly), then make change in that branch.
But thanks to let me know this. : ) I will track this change